server/server.js

```diff
@@ -9,6 +9,11 @@ import Shopify, { ApiVersion } from "@shopify/shopify-api";
 import Koa from "koa";
 import next from "next";
 import Router from "koa-router";
+import bodyParser from "koa-bodyparser";
+import { ShopModel } from "./db/models/shop";
+import { SessionModel } from "./db/models/session";
+
+import "./db/seed";
```

Comment:
This should definitely be a `yarn` script, I just wanted to get thoughts on using a db at all before looking into doing that!